api/src/main/java/org/pmiops/workbench/cohortbuilder/CohortBuilderServiceImpl.java

```diff
@@ -43,7 +44,27 @@
 import org.pmiops.workbench.cdr.model.DbCriteriaAttribute;
 import org.pmiops.workbench.cohortbuilder.mapper.CohortBuilderMapper;
 import org.pmiops.workbench.db.model.DbConceptSetConceptId;
-import org.pmiops.workbench.model.*;
+import org.pmiops.workbench.model.AgeType;
```

Comment:
We generally try to follow the Google Java Style Guide which includes "no wildcard imports" - my IntelliJ formatted this to do so automatically

Reply:
Need to use  .* since there are more than 5 classes that are imported. This is the google-style imports recommended by workbench and used by both IDE and gradle.spotlessJavaApply

![image](https://user-images.githubusercontent.com/4452480/183521766-652e2852-824c-496f-9eb8-567faa699c95.png)

Reply:
Thanks. Fixed my codeStyle for IntelliJ. Can you also get the 2 other files? CohortBuilderService and CohortBuilderController

Reply:
thank you